src/web/core/src/main/java/org/geoserver/web/publish/PublishedConfigurationPage.java

```diff
@@ -213,6 +226,12 @@ protected WebMarkupContainer newLink(String linkId, final int index) {
 
                             @Override
                             public void onSubmit() {
+                                WebMarkupContainer panel =

```

Comment:
I've put that code in case we have a layer group belonging to a ws that is modified to be a global one. Since the way taken has been to not delete or rename access data rule of a layer group when it is modified to or from being global, reloading Model permits to have on the security tab corresponding rules to layer group mode when it is modified